packages/api/package.json

```diff
@@ -4,12 +4,20 @@
   "license": "MIT",
   "main": "./index.ts",
   "types": "./index.ts",
+
   "scripts": {
     "check": "tsc --noEmit",
     "clean": "rm -rf .turbo node_modules",
-    "lint": "eslint . --ext .ts,.tsx",
+    "lint": "eslint .",
     "type-check": "tsc --noEmit"
   },
+  "eslintConfig": {
```

Comment:
Remove the per-project config (for now?) and replace with this config in the package.json for each package